Sources/OAuth2Client/NXOAuth2Request.m

```diff
@@ -116,6 +117,24 @@ - (void)performRequestWithSendingProgressHandler:(NXOAuth2ConnectionSendingProgr
     self.me = self;
```

Comment:
I am just replicating the form of the existing method `performRequestWithSendingProgressHandler`. This reference is removed on closures methods such as `cancel` or `oauthConnection:didFinishWithData:`. Anyway I've just refactored that to make it more clear.